scripts/gdprdeleteall.php

```diff
@@ -0,0 +1,42 @@
+#!/usr/local/bin/php -q
+<?php
+/**
+ * DO NOT RUN THIS SCRIPT WILLY NILLY
+ * this script will delete the personal data of any members that have been emailed regarding deletion and have not acted
+ * you should probably run the gdpremail script first then wait a week or two
+ * 
+ * run with "php gdprdeleteall.php"
```

Comment:
perhaps this now needs a slightly less scary name?

Reply:
I don't mind scaring people off if you don't know what you're doing you shouldn't be running the script